typescript/audio-summarizer/README.md

```diff
@@ -0,0 +1,188 @@
+# Emojifier 🤖🎨 - A simple audio summarizer with observability on Literal AI
+
+In this cookbook, we will start with a simple audio summarizer using the Web Speech API and Next.js. We will then add observability to the application using Literal AI.
+
+The base application converts audio to text and summarize the text as a sequence of emojis. It consists of :
+
+- A web page with a simple UI to record audio, using the [AudioRecorder](https://www.npmjs.com/package/react-audio-voice-recorder) React component.
+- A Next.js API route to convert the audio to text using OpenAI's [Whisper API](https://openai.com/index/whisper/).
+- A Next.js API route to summarize the text using OpenAI's [GPT-4](https://openai.com/index/gpt-4/).
+
+You can test the base application by running the development server. The only pre-requisite is to have Node.js installed on your machine.
+
+```bash
+cd without-literal
+npm install
+npm run dev
+```
+
+## Prerequisites
+
+Before we get started, you will need to create a free account on [Literal AI](https://cloud.getliteral.ai/). You will also need to create a new project and generate an API key from your project's "Settings > General" section.
+
+For the OpenAI calls to work, you will also need to generate an OpenAI API key from the [OpenAI API](https://platform.openai.com/api-keys) platform.
+
+You can now copy the provided `.env.example` file to a new `.env` file and fill in the required environment variables.
+
+## Adding Literal AI to the application
+
+We will start by adding the Literal AI SDK to the application. You can install the SDK using npm:
+
+```bash
+npm install @literalai/client
+```
+
+## Decoupling the prompt from our code
```

Comment:
I think it will be interesting to show the prompt to the user.

Even better, if you can use Literal AI's UI to show the prompt :)

Reply:
Maybe you can split this section into two:

### Creating prompts in Literal AI
### Retrieving prompts from Literal AI

Reply:
done